owl/SOMA-OBJ.owl

```diff
@@ -24,6 +24,31 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-OBJ.owl#hasShape -->
+
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/SOMA-OBJ.owl#hasShape">
```

Comment:
There is a nameclash: *hasShape* is already defined as a sub-property of *hasQuality*.

Would it be ok to name this property *hasShapeRegion*? Then the domain could be `(Shape or PhysicalObject)`.

Reply:
Sure!